hyperstyle/src/python/review/inspectors/ij/ij_inspections.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import os
+from pathlib import Path
+from typing import Any, Dict, List
+
+import requests
+
+from hyperstyle.src.python.review.common.language import Language
+from hyperstyle.src.python.review.inspectors.base_inspector import BaseInspector
+from hyperstyle.src.python.review.inspectors.ij.model import IJCode, IJInspectionResult
+from hyperstyle.src.python.review.inspectors.inspector_type import InspectorType
+from hyperstyle.src.python.review.inspectors.issue import BaseIssue
+
+LANGUAGE_TO_ID = {
+    Language.PYTHON: "Python",
+}
+
+CODE_SERVER_HOST = "CODE_SERVER_HOST"
+CODE_SERVER_PORT = "CODE_SERVER_PORT"
+CODE_SERVER_ROOT = "CODE_SERVER_ROOT"
+
+logger = logging.getLogger(__name__)
+
+
+class IJInspector(BaseInspector):
+    inspector_type = InspectorType.IJ
+
+    def __init__(self, language: Language):
+        self.host = os.environ.get(CODE_SERVER_HOST, "0.0.0.0")
+        self.port = os.environ.get(CODE_SERVER_PORT, 8080)
+        self.root = os.environ.get(CODE_SERVER_PORT, "code/server/api/v1/")
+        self.languageId = LANGUAGE_TO_ID[language]
+
+    def inspect(self, path: Path, config: Dict[str, Any]) -> List[BaseIssue]:
+        with open(path, "r") as code_file:
+            code = code_file.read()
```

Comment:
We already have a function for it - https://github.com/hyperskill/hyperstyle/blob/main/hyperstyle/src/python/review/common/file_system.py#L99